feature/experimental/tunnel/otg_tests/tunnel_interface_based_ipv6_gre_encapsulation_test/README.md

```diff
@@ -0,0 +1,97 @@
+# TUN-1.4: Interface based IPv6 GRE Encapsulation
+
+## Summary
+
+Validate Interface based Ipv6 GRE Tunnel Config.
+
+## Procedure
+Validate the GRE configuration
+GRE Tunnel interfaces configuration options
+Tunnel source
+Tunnel source should be able to configure with unnumbered interface address
+Tunnel Destination
+Directly on Tunnel interface or Tunnel Group
+Configure such 32 tunnel interfaces
+Filter match and action
```

Comment:
This being an interface based Tunnel (and not Filter based tunnel), these Filters shouldnt exist?

Reply:
Filter related steps and paths are removed.